catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/DownloadStatus.java

```diff
@@ -13,18 +13,26 @@
  */
 package ddf.catalog.resource.download;
 
-/**
- * The current status of a single product download. Since a product retrieval may go through several
- * retry attempts during the download process, it is possible for the download status to have
- * several of these values over the entire span of the download. For example, a product download
- * could be interrupted due to a brief network connection drop, and then the download could complete
- * successfully.
- */
-public enum DownloadStatus {
```

Comment:
This enum was moved to `ddf.catalog.resource.download.ReliableResourceStatus` because it's only ever used for `ReliableResourceStatus`s. The term `DownloadStatus` was kind of overloaded in these classes, but it should probably just refer to the map of details for a download status.